test/test_jit.py

```diff
@@ -1401,6 +1401,22 @@ def fn(x, y, z):
         ast = torch.jit.frontend.get_jit_ast(fn)
         self.assertExpected(str(ast))
 
+    def test_e2e_compilation(self):
+        def fn(x, y, z):
+            q = x + y - z
+            w = -z
+            m = y
+            if not x and not y and z:
+                m = x
+            while x < y > z:
+                q = x
+            return x
+
+        ast = torch.jit.frontend.get_jit_ast(fn)
+        cu = torch.jit._jit_script_compile('')
+        cu.define_function(ast)
+        self.assertExpected(str(cu.get_graph('fn')))
```

Comment:
I think you can just replace the previous test with this one. BTW my PR adds an e2e test that also runs the created graph